nodejs_waiter/src/waiter.rs

```diff
@@ -4,7 +4,7 @@ use holochain_core::{
     nucleus::ZomeFnCall,
     signal::{Signal, SignalReceiver},
 };
-use holochain_core_types::entry::Entry;
+use holochain_core_types::{cas::content::AddressableContent, entry::Entry};
```

Comment:
why does this come in?

Reply:
If you use `.address()` on something that implements AddressableContent, you also have to include the trait in scope.

Reply:
I see

Reply:
Yeah here https://github.com/holochain/holochain-rust/pull/1072/files/8ddb359fbe20ed4521311e00b3d51376c835fbcc#diff-f6e488476c3a39ed86c9080bee61ef9aR209